includes/admin/tools/class-edd-tools-recount-all-stats.php

```diff
@@ -13,22 +13,39 @@
 // Exit if accessed directly
 defined( 'ABSPATH' ) || exit;
 
-if ( ! class_exists( 'EDD_Tools_Recount_Download_Stats' ) ) {
-	require_once EDD_PLUGIN_DIR . 'includes/admin/tools/class-edd-tools-recount-download-stats.php';
-}
-
 /**
  * EDD_Tools_Recount_All_Stats Class
  *
  * @since 2.5
  */
-class EDD_Tools_Recount_All_Stats extends EDD_Tools_Recount_Download_Stats {
+class EDD_Tools_Recount_All_Stats extends EDD_Batch_Export {
```

Comment:
With the helper function, this class doesn't need to extend the `EDD_Tools_Recount_Download_Stats` class anymore--more simplification.